api/internal/git/gitrunner.go

```diff
@@ -46,9 +46,9 @@ func (r gitRunner) run(args ...string) error {
 		cmd.String(),
 		r.duration,
 		func() error {
-			out, err := cmd.CombinedOutput()
+			_, err := cmd.CombinedOutput()
 			if err != nil {
-				return errors.Wrapf(err, "failed to run '%s': %s", cmd.String(), string(out))
+				return errors.Wrapf(err, "git cmd = '%s'", cmd.String())
```

Comment:
Same here: I'm fine with these changes staying if you want; it's the api/internal/target/kusttarget.go changes specifically that I think need dedicated attention in a new PR. Sorry if that was unclear.